packages/empty-state/src/BasicEmptyState/BasicEmptyState.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import Button from '@leafygreen-ui/button';
+import { Link } from '@leafygreen-ui/typography';
+
+import { BasicEmptyState } from '.';
+
+beforeEach(() => {
+  // silence all console errors
+  jest.spyOn(console, 'error').mockImplementation(() => {});
+});
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('packages/empty-state/basic', () => {
+  test('renders title and description', () => {
+    const { getByText } = render(
+      <BasicEmptyState title="test title" description="test description" />,
+    );
+    expect(getByText('test title')).toBeInTheDocument();
+    expect(getByText('test description')).toBeInTheDocument();
+  });
+  test('renders primary button', () => {
+    const { getByText } = render(
+      <BasicEmptyState
+        title="test title"
+        description="test description"
+        PrimaryButton={<Button>test button</Button>}
```

Comment:
Correct me if I'm wrong but I believe for Button components we accept leftGlyph and rightGlyph props that expect elements but are not Pascal Case - I think we should be doing the same here